packages/cms-cli/commands/secrets.js

```diff
@@ -95,8 +100,39 @@ function configureSecretsDeleteCommand(program) {
   addConfigOptions(program);
 }
 
+function configureSecretsListCommand(program) {
+  program
+    .version(version)
+    .description('List all HubSpot secrets')
+    .action(async (options = {}) => {
```

Comment:
The last parameter passed to action is always the commander instance.
Thus `program === options`.  Given this I've been preferring to use `program`.

For example: https://github.com/HubSpot/hubspot-cms-tools/blob/master/packages/cms-cli/commands/fetch.js#L32-L39